app/containers/UnlockEtherContainer.tsx

```diff
@@ -0,0 +1,200 @@
+import { TokenType } from "../actions/constants";
+import { LoadingIndicator } from "../components/LoadingIndicator";
+import MoneyComponent from "../components/MoneyComponent";
+import { TextCopyable } from "../components/TextCopyable";
+import { selectLoading, selectUnlockDateEth } from "../reducers/aftermathState";
+import { IAppState } from "../reducers/index";
+import { etherLock, etherToken, neumark } from "../web3/contracts/ContractsRepository";
+
+import { calculateValueAfterPenalty } from "../web3/utils";
+import * as styles from "./Aftermath.scss";
+
+const GAS_LIMIT = 600000;
+
+interface IUnlockEtherContainerProps {
+  params: { address: string };
+  loadAftermathDetails: (address: string) => any;
+  isLoading: boolean;
+  lockedAmountEth: string;
+  neumarkBalance: string;
+  neumarkNeededToUnlockEth: string;
+  unlockDateEth: Moment;
+  penaltyFractionEth: string;
+}
+
+class UnlockEtherContainer extends React.Component<IUnlockEtherContainerProps> {
+  public componentDidMount() {
+    this.props.loadAftermathDetails(this.props.params.address);
+  }
+
+  public render() {
+    if (this.props.isLoading) {
+      return <LoadingIndicator />;
+    }
+
+    return (
+      <div>
+        <h1>Unlock your funds</h1>
+        <p>You can unlock your funds anytime you want.</p>
+        <p>
+          You need to give back all neumarks rewarded to you. If you unlock before unlock period is
+          over you will be penalized with 10% value of your funds.
+        </p>
+        <div className={styles.section}>
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your wallet address:</div>
+            <div className={styles.value}>
+              {this.props.params.address}
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your NEU balance: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.neumarkBalance} tokenType={TokenType.NEU} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Ether locked: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.lockedAmountEth} tokenType={TokenType.ETHER} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>NEU needed to unlock ether funds: </div>
+            <div className={styles.value}>
+              <MoneyComponent
+                value={this.props.neumarkNeededToUnlockEth}
+                tokenType={TokenType.NEU}
+              />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Free unlock date: </div>
+            <div className={styles.value}>
+              {this.props.unlockDateEth.format("YYYY-MM-DD")}
+            </div>
+          </div>
```

Comment:
we need another infoBox which will show amount of the penalty
`Unlock fee distributed back to remaining NEU holders:`
This one should be bigger/bold